src/jit/compiler.cpp

```diff
@@ -11097,3 +11097,831 @@ bool Compiler::killGCRefs(GenTree* tree)
 
     return false;
 }
+
+#ifdef DEBUG
+//------------------------------------------------------------------------
+//                      VariableLiveRanges dumpers
+//------------------------------------------------------------------------
+
+// Dump "VariableLiveRange" when code has not been generated and we don't have so the assembly native offset
+// but at least "emitLocation"s and "siVarLoc"
+void dumpVariableLiveRange(const VariableLiveRange* varLiveRange, const CodeGenInterface* codeGen)
+{
+    // Check pointers parameter are not nullptr
+    noway_assert(codeGen != nullptr);
```

Comment:
This is debug code, `assert` would be more than sufficient. In fact these 2 should probably be deleted. You'll crash in the code below if either is `nullptr`.

Reply:
I agree with you that you would fail immediately below if codeGen is nullptr. It doesn't happen the same with emit, it would happen in emitLocation.